server/appsettings.Development.json

```diff
@@ -22,6 +22,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": ""
+    "DefaultConnection": "Initial Catalog=OfficeBookingDB; Data Source=localhost,1450; Persist Security Info=True;User ID=SA;Password= 2Secure*Password2; TrustServerCertificate=True"
```

Comment:
This should be left blank, and you should use your user secrets to hold it as the local config can differ between developer environments.

Reply:
we have only one developer environment and in the future according to Marius we should delete the environment environment in azure and only keep the local development. So it is not an issues of multi developer environments. If we shall the user secrets between developers then we should have a place to document it, so that the new developer know how to set it up. The password here is just what we set up the Docker, it is not sensitive info.  instructions for configuring user secrets in the README unless there are any objections.